stable/prometheus-operator/Chart.yaml

```diff
@@ -10,7 +10,7 @@ name: prometheus-operator
 sources:
   - https://github.com/coreos/prometheus-operator
   - https://coreos.com/operators/prometheus
-version: 6.12.1
+version: 6.12.2
```

Comment:
[Since this is not a bugfix but an enhancement, the minor version should be increased. i.e. `6.13.0` ](https://semver.org/#spec-item-7)